pages/common/vim.md

```diff
@@ -14,6 +14,28 @@
 
 `view {{file}}`
 
-- Exit vim:
+- Switch to normal mode:
 
-`[Esc] (to switch to normal mode), then :q`
+`<Esc>`
+
+- Switch to insert mode from normal mode:
+
+`i`
+
+- Run a command mode from normal mode:
+
+`:{{command}}<Cr>`
+
+- Common commands:
```

Comment:
We sholdn't try to cram too much of modal editing into this page. I'd rather present the most common examples individually, and let them climb [vim's learning curve](http://stackoverflow.com/questions/10942008/what-does-emacs-learning-curve-actually-look-like) at their own pace.